src/components/nodes/Number.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="q-px-md q-pt-md">
```

Comment:
This does not have a bottom margin. Use the `q-py-md` class instead of `q-pt-md`.